packages/lib-classifier/src/store/utils/session.spec.js

```diff
@@ -87,7 +87,7 @@ describe('Store utils > sessionUtils', function () {
 
     it('should call generateSessionID', function () {
       sessionUtils.getSessionID()
-      expect(generateSessionIDStub.calledTwice).to.be.true
+      expect(generateSessionIDStub.calledTwice).to.be.true()
```

Comment:
This should change to `expect(generateSessionIDStub).to.have.been.calledTwice()`

Reply:
This is an odd test. I wonder why the session ID is generated twice?